rfp-proposals/ars-go-filecoin-storage-helper.md

```diff
@@ -0,0 +1,99 @@
+# RFP Proposal: `ars-go-filecoin-storage-helper`
+
+**Name of Project:** go-filecoin-storage-helper
+
+**Link to RFP:** [rfp-storage-helper-libraries](https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-storage-helper-libraries.md)
+
+**RFP Category:** `devtools-libraries`
+
+**Proposer:** [@waynewyang](https://github.com/waynewyang)
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:**  "Yes"
+
+# Project Description
+
+The project is based on [rfp-storage-helper-libraries](https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-storage-helper-libraries.md), We think we're  very clear about the project want to do, our implementation will meet the need of [RFP](https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-storage-helper-libraries.md),
+
+We think the keypoint is that client must pay for the metadata, the metadata is used for data retrieval,so the design of the metadata is very important,now we already have the idea.
+
+Besides,we have another idea we'll provide the function in storage helper that metadata encryption.
+
```

Comment:
Can you describe how you expect this library to be used? Does it have a command-line interface? If so, what are some of the commands? Can we make it work with lotus?

Reply:
- You can use by import the package or use command, you can view the workflow [here](https://github.com/arsyun/go-filecoin-storage-helper)
- Yes, such as 
    - storage-helper store [path]
        - --status [cid] // check the storage flowwork status
        - --expire [time] // set expire time
        - --price [FIL] // set price
        - ......
    - storage-helper retrieval
        - ......
- We'll support lotus although lotus was not open source when the proposal was submitted

Reply:
- You can use by import the package or use command, you can view the workflow [here](https://github.com/arsyun/go-filecoin-storage-helper)
- Yes, such as 
    - storage-helper store [path]
        - --status [cid] // check the storage flowwork status
        - --expire [time] // set expire time
        - --price [FIL] // set price
        - ......
    - storage-helper retrieval
        - ......
- We'll support lotus although lotus was not open source when the proposal was submitted